bin/kano-login

```diff
@@ -87,10 +88,10 @@ class MainWindow(ApplicationWindow):
         apply_styling_to_screen(css_path)
 
         if screen == "register":
-            # Gives the user the option to log in or register
             FirstScreen(self)
         elif screen == "login":
```

Comment:
Could we move the `login` logic into a class method? Feel a bit uncomfortable doing all this stuff in the constructor...